lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -445,20 +445,39 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   }
 
   // Add social share bar.
+  $custom_social_copy_variables = [
+    'custom_facebook_copy' => $campaign->variables['custom_facebook_copy'],
+    'custom_twitter_copy' => $campaign->variables['custom_twitter_copy'],
+    'custom_tumblr_copy' => $campaign->variables['custom_tumblr_copy'],
+  ];
+
+  foreach ($custom_social_copy_variables as $key => $value) {
+    if ($value) {
+      $$key = $value;
+    }
+    elseif (isset($campaign->fact_problem['fact'])) {
```

Comment:
in what situations is this set? could it not be added to the `$custom_social_copy_variables` above?